### DISCUSSION TITLE
Appium can't Identify the correct element in split screen .

### DISCUSSION THREAD

**Discussion by LiHongyuan1** [appium/appium#17285] [General]
Background :  working for vehicle automation . when android screen open split screen . we found that Appium can't Identify the correct element in split screen . 
![image](https://user-images.githubusercontent.com/55521638/181709238-80c63ded-291d-4c30-82db-43421e086339.png)

vehicle android  version:  8.8
Appium server GUI : 1.22.0     Appium Inspector:  App version: 2021.9.2  Electron:13.5.0  Node.js :14.16.0
Desired Capabilities:
{
  "platformName": "Android",
  "appium:deviceName": "FordP",
  "appium:appPackage": "com.baidu.xiaoduos.launcher",
  "appium:appActivity": ".ui.MainActivity",
  "automationName": "UiAutomator2",
  "enableMultiWindows": true,
  "appium:noReset": true,
  "appium:unicodeKeyboard": true,
  "appium:resetKeyboard": true
}

Description : in Desired Capabilities have add enableMultiWindows parameter .  when click split screen  Radio element  but  found is Main screen IQIYI element . the element is not right . 

question :   
does android  8.8 not support for Multi Windows or appium not support for this scenario ?

**Comment by KazuCocoa** [comment]
> Description : in Desired Capabilities have add enableMultiWindows parameter . when click split screen Radio element but found is Main screen IQIYI element . the element is not right .

Is it via script (I mean not via Appium Inspector)? If Appium could get the element via find_element, but the click was wrong, maybe something is wrong in the Android internal. Potentially you need to manually tweak the coordinate and click a point with the coordinate base with W3C actions.

>  does android 8.8 not support for Multi Windows or appium not support for this scenario ?

It depends on the OS/Accessibility. If something was wrong, maybe Appium does not work well since it means probably AccessibillityNodeInfo, Appium gets element info via it, does not work well.
You could also try out appium espresso driver.
